docs/walletkit/android/experimental/chain-abstraction.mdx

```diff
@@ -10,11 +10,11 @@ For example, when an app requests a 100 USDC payment on Base network but the use
 Apps need to pass `gas` as null, while sending a transaction to allow proper gas estimation by the wallet. Refer to this [guide](../../../appkit/next/experimental/chain-abstraction.mdx) for more details.
 :::
 
-When a app requests a transaction through `eth_sendTransaction`, you need to:
-1. Detect if chain abstraction is needed
-2. Generate necessary bridging transactions
-3. Handle transaction execution and monitoring
-4. Complete the original transaction once bridging is done
+When an app sends a transaction request to your wallet, you need to:
```

Comment:
Why app? When a user wants to send stable coin in a wallet

Reply:
Updated